ui/ui.xml

```diff
@@ -0,0 +1,178 @@
+<html>
+  <head>
+    <title>{{{filename}}}</title>
+    <link href="resources/css/reset.css" rel="stylesheet" type="text/css" />
+    <link href="resources/css/960.css" rel="stylesheet" type="text/css" />
+    <link href="https://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.8.2/css/jquery.dataTables.css" rel="stylesheet" type="text/css" />
+    <link href="resources/css/jquery-ui-1.8.21.custom.css" rel="stylesheet" type="text/css" />
+    <link href="resources/css/kif.css" rel="stylesheet" type="text/css" />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js" type="text/javascript"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.8.18/jquery-ui.min.js" type="text/javascript"></script>
+    <script src="https://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.8.2/jquery.dataTables.min.js" type="text/javascript"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/0.7.0/mustache.min.js" type="text/javascript"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.4.2/underscore-min.js" type="text/javascript"></script>
+    <script src="resources/js/jquery.tooltip.min.js" type="text/javascript"></script>
+    <script src="resources/js/jquery.zclip.min.js" type="text/javascript"></script>
+    <script src="resources/js/json2.js" type="text/javascript"></script>
+    <script src="resources/js/kif.js" type="text/javascript"></script>
+  </head>
+
+  <body>
+    <span id="ticket-info" style="display: none">
+      <div id="ticket-info-map">{{{ticket-info-json}}}</div>
```

Comment:
Just to double-check: does mustache/clostache support variable names with hyphens? A lot of stuff is stupid about that, but I'm assuming you tested.